B/blis/common.jl

```diff
@@ -87,10 +87,6 @@ function blis_script(;blis32::Bool=false)
         # Unscreen Arm SVE code for metaconfig.
         patch kernels/armsve/bli_kernels_armsve.h \
             < ${WORKSPACE}/srcdir/patches/armsve_kernels_unscreen_arm_sve_h.patch
-        patch kernels/armsve/1m/old/bli_dpackm_armsve512_int_12xk.c \
-            < ${WORKSPACE}/srcdir/patches/armsve_kernels_unscreen_arm_sve_h.patch
-        patch kernels/armsve/1m/bli_dpackm_armsve256_int_8xk.c \
-            < ${WORKSPACE}/srcdir/patches/armsve_kernels_unscreen_arm_sve_h.patch
```

Comment:
Why are these removed?  If they are unneeded, should the corresponding file be removed?  But I hope we aren't missing optimisations on ARM SVE architectures.

Reply:
The second (`bli_dpackm_armsve256_int_8xk.c`) doesn't exist anyone after a refactor.

Reply:
However the applied `armsve_kernels_unscreen_arm_sve_h.patch` is still used in the patch in line 89.

Reply:
The `bli_dpackm_armsve512_int_12xk` still appears to have the `!defined(BLIS_FAMILY_A64FX)` in it, but is that actually used anywhere? A search in the BLIS code base for the symbol in that kernel (`bli_dpackm_armsve512_int_12xk`) does not show any other uses.

The `bli_dpackm_armsve256_int_8xk.c` appears to have been changed, there is only a `bli_dpackm_armsve256_int_8x10.c` now, but it does have the `!defined(BLIS_FAMILY_A64FX)` guarding it, however that kernel is actually also only selected part of the time, since it is configured with this case (https://github.com/flame/blis/blob/6d0ab74f6975fdf4d19cee06d946b09b6ca89656/kernels/armsve/bli_kernels_armsve.h#L47C1-L51C57):

```
// Use SVE intrinsics only for referred cases.
#if !defined(BLIS_FAMILY_A64FX)
PACKM_KER_PROT( double,   d, packm_armsve256_int_8x10 )
#endif
PACKM_KER_PROT( double,   d, packm_armsve512_asm_16x10 )
```
So, will this still be a problem? It looks like they have an SVE 512 assembly kernel for the A64FX processor and so won't be using that SVE 256 kernel there.

Reply:
The first (`patch kernels/armsve/1m/old/bli_dpackm_armsve512_int_12xk.c`) seems to be unused in the code anyway.
cf. https://github.com/flame/blis/commit/2604f4071300d109f28c8438be845aeaf3ec44e4

Reply:
@xrq-phys Do you have insights here on what ARM SVE optimisations?